xbmc/network/httprequesthandler/HTTPImageHandler.cpp

```diff
@@ -37,7 +37,25 @@ CHTTPImageHandler::CHTTPImageHandler(const HTTPRequest &request)
     XFILE::CImageFile imageFile;
     const CURL pathToUrl(file);
     if (imageFile.Exists(pathToUrl))
+    {
       responseStatus = MHD_HTTP_OK;
+      struct __stat64 statBuffer;
+      if (imageFile.Stat(pathToUrl, &statBuffer) == 0)
+      {
+        struct tm *lastChange;
+#ifdef HAVE_LOCALTIME_R
```

Comment:
is [that](https://github.com/xbmc/xbmc/blob/master/project/cmake/scripts/common/ArchSetup.cmake#L96) the probe you want?